transifex/native/cds.py

```diff
@@ -220,6 +220,34 @@ def push_source_strings(self, strings, purge=False):
 
         return response
 
+    def get_push_status(self, job_path):
+        """Get source string push job status
+
+        :param str job_path: Job url path
+        :return: the HTTP response object
+        :rtype: requests.Response
+        """
+        if not self.secret:
+            raise Exception('You need to use `TRANSIFEX_SECRET` when polling '
+                            'source content push')
+
+        try:
+            response = requests.get(
+                self.host + job_path,
+                headers=self._get_headers(use_secret=True),
+            )
+            response.raise_for_status()
+
+        except requests.ConnectionError:
+            logger.error(
+                'Error polling source strings push to CDS: ConnectionError')
+        except Exception as e:
+            logger.error(
+                'Error polling source strings push to CDS: UnknownError '
+                '(`{}`)'.format(str(e)))
```

Comment:
Minor: Why differentiate between the two exception types? Wouldn't more details about the exception be useful for the logs?

Reply:
Pure copy paste from the `push_source_strings` code :P